test/spec/file-utils.spec.js

```diff
@@ -87,180 +87,188 @@ describe("fileUtils", () => {
             lodash: "1.0.0"
           }
         });
+      });
+    });
+
+    it("should reject on a JSON parse error", () => {
+      mockfs({
+        "package.json": "bad"
+      });
 
-        fs.readFile.restore();
+      return fileUtils.readPackage().catch((err) => {
+        expect(err).to.be.an.instanceOf(SyntaxError);
       });
     });
 
     it("should reject on read error", () => {
-      sinon.stub(fs, "readFile", (filePath, opts, cb) => cb("mock error"));
+      mockfs({});
 
       return fileUtils.readPackage().catch((err) => {
-        expect(err).to.equal("mock error");
-
-        fs.readFile.restore();
+        expect(err.code).to.equal("ENOENT");
       });
     });
   });
 
   describe("removeFile", () => {
     it("should not throw without a remove file error", () => {
-      sinon.stub(fs, "unlink", (filePath, cb) => cb());
-
-      return fileUtils.removeFile("remove.js").then((result) => {
-        expect(result).to.equal(undefined);
-
-        fs.unlink.restore();
+      mockfs({
+        "remove.js": "mock contents"
       });
+
+      return fileUtils
+        .removeFile("remove.js")
+        .then(() => testHelpers.fileExists("remove.js"))
+        .then((fileExists) => expect(fileExists).to.be.false);
```

Comment:
What's our thought on explicit braces here to make clear we don't intend to return the result?